hazelcast/src/main/java/com/hazelcast/cache/impl/CacheEntryProcessorEntry.java

```diff
@@ -141,6 +141,10 @@ protected V getRecordValue(R record) {
         return (V) objValue;
     }
 
+    public R getRecord() {
```

Comment:
maybe we can add an assertion here to be sure that this method is called by partition threads `assert !(Thread.currentThread() instanceof PartitionOperationThread)`